loom/engine/bindings/loom/lmApplicationTasks.cpp

```diff
@@ -33,10 +33,14 @@ extern "C"
 
 void loom_tick()
 {
+    LSLuaState *vm = NULL;
```

Comment:
It makes it not break if there's no vm available (edge case, but if you disable reinitialization after reloading, this can be the case).